ball.py

```diff
@@ -0,0 +1,80 @@
+import pygame
+from random import random
+
+
+class Ball:
+
+    def __init__(self, position, radius):
+        self.gameDisplay = pygame.display.get_surface()
+
+        self.radius = radius
+        self.position = pygame.Vector2(position[0], position[1])
+        self.velocity = pygame.Vector2(7, -7)
+
+        self.color = (0, 0, 0)
+
+    def set_velocity(self, velocity):
+        self.velocity = velocity
+
+    def update(self):
+        self.position[0] += self.velocity[0]
+        self.position[1] += self.velocity[1]
+
+        bounce_padding = 10
+
+        x_min = 0 + bounce_padding
+        x_max = self.gameDisplay.get_width() - self.radius * 2 - bounce_padding
+        y_min = 0 + bounce_padding
+        y_max = self.gameDisplay.get_height() - self.radius * 2 - bounce_padding
```

Comment:
0 + bounce_padding?

Reply:
It appeared to me more readable because in everything else I was adding the bounce padding to the coordinates